modules/upgrade/pages/migrate-4x-4x.adoc

```diff
@@ -1,32 +1,46 @@
 [[bp.sp.migration.version]]
-= Migrating from Version 4 to 4.x
+= Upgrading from Version 4 to 4.x
 
-Migrating {productname} from version 4 to 4.x (when available) can be done using either the {yast} Online Migration tool, or with the Zypper command line tool.
+You can upgrade {productname} from version 4 to 4.x (when available) using either the {yast} online migration tool or the Zypper command line tool.
```

Comment:
Why are we renaming this from `migration` to `update` if in the end we use `yast2-migration`?

Reply:
Yes, this is not nice.  But we said we want to call the small updates (z level) such as package updates or MUs _updates_, the y level updates (4.0.z -> 4.1.0) _upgrades_, and the x level updates (3.y.z -> 4.0.0) _migrations_.

Unfortunately, the y level updates are also known as "SP migration"...  @Loquacity do we have to improve our terminology?

Reply:
Oh, probably. But there's no consensus anywhere on this. If the system or the UI was consistent, we could use those terms, but they aren't. So my vote is to standardise within our docs and hope that everything else catches up.

Reply:
Never give up :)  We must pay attention when they start changing something in that area and and propose our ideas as early as possible.